routes/user/presentations/presentation/handlers.js

```diff
@@ -331,180 +330,53 @@ var getPresentationStats = gen.lift(function *getPresentationStats(req, res, nex
   
 });
 
-var getConf = coroutine(function* getConf(conf) {
-  var slideConf = [];
-  for (var key in conf) {
-    if (conf.hasOwnProperty(key)) {
-      // input: select
-      if ( key == "slideflow" || key == "assessment") {
-        var type = "select";
-        var options;
-        if ( key == "slideflow" ) options = slideflowTypes; 
-        if ( key == "assessment" ) options = assessmentTypes;         
-   
-        var newOptions = [];
-        for ( var i=0; i<options.length; i++ ) {
-          newOptions.push({
-            option   : options[i],
-            selected : options[i] == conf[key]
-          });
-        }
-       
-        slideConf.push({
-          id: key.toLowerCase(),
-          key: key,
-          type: type,
-          value: null,
-          options: newOptions
-        })
-      }
-      // input: number
-      if ( key == "maxNumSubmissions" ) {
-        var type = "number";
-        slideConf.push({
-          id: key.toLowerCase(),
-          key: key,
-          type: type,
-          value: conf[key]
-        })
-      }
-    }
-  }
-
-  return slideConf
-});
-
-
-var transform = coroutine(function* transform(slides) {
-  var data = [];
-  for (var key in slides) {
-    if (slides.hasOwnProperty(key)) {
-      var slide = {
-        index: key,
-        exercises: []
-      };
-      
-      for ( var i=0; i<slides[key].length; i++ ) {
-        var exObject = yield Exercise.findById(slides[key][i]).exec();
-        var exercise = {};
-        exercise.uid = slides[key][i]; 
-        exercise.names = ['maxNumSubmissions', 'confidence'];
-        exercise.maxNumSubmissions = exObject.maxNumSubmissions;
-        exercise.confidence = exObject.confidence;
-        slide.exercises.push(exercise);
-      }
-      data.push(slide)
-    }
-  }
-  return data.reverse();
-});
-
-var isSlideshowActiveByUser = coroutine(function* isSlideshowActiveByUser(slideshowId, userId) {
-  var query = {
-    'presenter': userId,
-    'slides': slideshowId,
-    'endDate': null
-  }
-  var sesstions = yield Session.find(query).exec();
-  if ( !sesstions || sesstions.length == 0 ) {
-    return { flag: false }
-  }
-  return { flag: true, sessions: sesstions }
-});
+var getPresentationSettings = coroutine(function* getPresentationSettingsGen(req, res) {
 
+  logger.debug('Get presentation settings from ' + req.user.username + ' with presentationId: ' + req.params.presentationId);
```

Comment:
try to use logger this way:
https://github.com/ASQ-USI/ASQ/blob/devel/routes/user/presentations/handlers.js#L141
having the variables in their own fields